### ISSUE TITLE
batch_size & dimension mismatch

### ISSUE THREAD

**Issue by o0o0o00o0** [M-Candy77/DyGait#1]
When I ran this project, I found some dimension mismatch problems, such as line 211 in dygait.py, which was already done in head0. After I solved these problems, I ran it with a batch_size of 8*4 and found softmax_accuracy standing still until the milestone reduced the learning rate. Is it because the batch_size is reduced, or what modifications do I need to make to reproduce it?
